inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -51,11 +51,9 @@
                     {% endif %}
                   </b>
                 </h4>
-                {% if record.authors %}
```

Comment:
I changed this and the general_macros file due to the changes I made in the collaboration field so as not to break it. Do you think we should move it to another PR?